app/views/templates/ssh/module_action.erb

```diff
@@ -29,6 +29,7 @@ template_inputs:
   input_type: user
   required: false
   advanced: true
+feature: :katello_module_stream_action
```

Comment:
@adamruzicka, just curious, after I've updated my local checkout and ran db:seed it failed with `... error (Psych::DisallowedClass): Tried to load unspecified class: Symbol`. Is `katello_module_stream_action` should be specified as a symbol?

Reply:
Wonder why I didn't see this. https://github.com/theforeman/foreman_remote_execution/pull/634